jscomp/core/j.ml

```diff
@@ -263,7 +263,7 @@ and statement_desc =
       * for_direction
       * block
       * Js_closure.t
-  | Continue of label
+  | Continue of label option
```

Comment:
I've noticed that the label is not used for ReScript at all. Maybe I should remove it completely?

Reply:
@cknitt @zth What do you think?

Reply:
Didn't know that "continue with label" exists in JS. :-)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/continue#using_continue_with_a_label

Usage of `label` is currently commented out here:
https://github.com/rescript-lang/rescript-compiler/blob/9b00a0eeedff2db41d0559fbb3530f66919dcc5d/jscomp/core/lam_compile.ml#L366

I don't know if we might want to use this feature in the future.
@cristianoc What do you think?

Reply:
Feels like we could just add it back if we end up needing it. I didn't know it existed in JS either.